src/components/carousel/daikin-carousel.ts

```diff
@@ -0,0 +1,353 @@
+    // it is determined to be an erroneous operation and the process is terminated.
+    if (Math.abs(result) < 10) {
+      return;
+    }
+
+    this._changeVisibleItem(Math.sign(result) as 1 | -1);
+
+    resetCoordinate();
+  }
+
+  override render() {
+    return html`<div
+      class="flex justify-center items-center flex-col gap-8"
+      style=${`--translate-transition-duration:${this.duration}ms;`}
+    >
+      <div class="flex justify-center items-center w-full gap-4">
+        <daikin-icon-button
+          ${ref(this._prevButton)}
+          variant=${this.controlButtonVariant}
+          color="neutral"
+          button-aria-label="Prev"
+          ?disabled=${this.currentIndex <= 0}
+          @click=${() => this._changeVisibleItem(-1)}
+        >
+          <span class=${cvaButton({ position: "left" })}></span>
+        </daikin-icon-button>
+        <div
+          class="w-full overflow-clip relative focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-ddt-color-common-border-focus"
+          aria-live="polite"
+          role="list"
+          tabindex="0"
+          @click=${this._emitCarouselClick}
+          @keydown=${this._handleKeydownCarousel}
+          @mousedown=${this._handleMousedown}
+          @mouseup=${this._handleMouseup}
+          @touchstart=${this._handleTouchstart}
+          @touchmove=${this._handleTouchmove}
+          @touchend=${this._handleTouchend}
+        >
+          <div
+            class="flex w-[--default-width] transition-transform translate-x-[--translate-x-width] duration-[--translate-transition-duration]"
+            part="carousel-items-container"
+            style=${ifDefined(
+              this.animation === "slide"
+                ? `--default-width:calc(100% * ${this._itemLength}); --translate-x-width:calc(-1 * 100% / ${this._itemLength} * ${this.currentIndex});`
+                : undefined
+            )}
+          >
+            <slot></slot>
+          </div>
+        </div>
+        <daikin-icon-button
+          ${ref(this._nextButton)}
+          variant=${this.controlButtonVariant}
+          color="neutral"
+          button-aria-label="Next"
+          ?disabled=${this.currentIndex >= this._itemLength - 1}
+          @click=${() => this._changeVisibleItem(1)}
+        >
+          <span class=${cvaButton({ position: "right" })}></span>
+        </daikin-icon-button>
+      </div>
+      <div class="flex gap-3">
+        <div
+          ${ref(this._indicatorButtons)}
+          class="flex items-center gap-4"
+          role="tablist"
+          @keydown=${this._handleKeydownIndicator}
+        >
+          ${repeat(
+            this._items,
+            (_, index) => index,
+            (_, index) =>
+              html`<button
+                class=${INDICATOR_CLASS_NAME}
+                aria-label="Slide ${index + 1}"
+                aria-selected=${`${this.currentIndex === index}` as const}
```

Comment:
I think we can do this

```suggestion
                aria-selected=${this.currentIndex === index}
```